src/react-apps/applications/runtime/src/shared/resources/attachments/fetch/fetchAttachmentsSagas.ts

```diff
@@ -15,9 +15,8 @@ export function* fetchAttachments(): SagaIterator {
   try {
     const altinnWindow: IAltinnWindow = window as Window as IAltinnWindow;
     const { org, app, instanceId } = altinnWindow;
-    const appId = `${org}/${app}`;
-    const attachmentListUrl = `${altinnWindow.location.origin}/${appId}/api/attachment/` +
-    `${instanceId}/GetFormAttachments`;
+    // tslint:disable-next-line:max-line-length
+    const attachmentListUrl = `${altinnWindow.location.origin}/${org}/${app}/api/attachment/${instanceId}/GetFormAttachments`;
```

Comment:
Is this url used anymore?

Reply:
maybe not, but we need to merge in this pr now. I can do a cleanup afterwards.